test_create_garden.py

```diff
@@ -0,0 +1,113 @@
+import unittest
+from main import create_garden, counting_items, plant, day_increment, grow_garden
+
+
+class Garden(unittest.TestCase):
+    def setUp(self):
+        self.garden1 = [
+            ["W", ".", ".", ".", ".", ".", ".", ".", ".", "F"],
+            ["F", ".", "W", ".", ".", "W", ".", ".", "F"],
+            [".", "W", ".", ".", ".", ".", ".", "W", "."],
+            [".", "F", ".", ".", ".", ".", ".", ".", ".", "."],
+            [".", "F", ".", ".", ".", ".", ".", ".", "F"],
+            [".", ".", "F", ".", ".", "W", "F", ".", "F"],
+            [".", ".", "W", ".", ".", "W", ".", ".", "."],
+        ]
```

Comment:
It would be best if you did not number your variables. Variables names should be descriptive.